docs/source/posts/2024/2024-08-15_week_9.rst

```diff
@@ -0,0 +1,63 @@
+Week 9: Hosting FineTuned Models
+================================
+
+.. post:: Aug 15 2024
```

Comment:
Should be 29th July based on week 9, but any date in the week is fine too. This will help render the blog sequence better, as Serge mentioned